subprojects/docs/src/docs/design/gradle-module-metadata-latest-specification.md

```diff
@@ -119,11 +119,13 @@ Existing metadata must remain compatible and thus tools supporting the Gradle Mo
 
 This value, nested in `variants`, must contain an object with the following values:
 
-- `url`: The location of the metadata file that describes the variant. A string. In version 1.0, this must be a path relative to the module.
+- `url`: The location of the metadata file that describes the variant. A string. In version 1.0 and 1.1, this must be a path relative to the module.
```

Comment:
```suggestion
- `url`: The location of the metadata file that describes the variant. A string. Since version 1.0, this must be a path relative to the module.
```

To have this future proof should there be a next version of the spec.

Reply:
I prefer being explicit and I do not think we have to do a new version for the changes reported here.